src/Compilation.zig

```diff
@@ -2584,6 +2584,10 @@ fn addNonIncrementalStuffToCacheManifest(
     man.hash.addOptional(opts.initial_memory);
     man.hash.addOptional(opts.max_memory);
     man.hash.addOptional(opts.global_base);
+    man.hash.add(opts.export_symbol_names.len);
+    for (opts.export_symbol_names) |symbol_name| {
+        man.hash.addBytes(symbol_name);
+    }
```

Comment:
```suggestion
    man.hash.addListOfBytes(opts.export_symbol_names);
```

Reply:
ah of course. I'll make the same changes to the place this was copy+pasted from